src/features/farming/animals/components/Barn.tsx

```diff
@@ -22,7 +22,7 @@ export const Barn: React.FC = () => {
 
   const openBarn = () => {
     setIsOpen(true);
-    barnAudio.play();
+    !barnAudio.playing() ? barnAudio.play() : undefined;
```

Comment:
I think this would read better using an `if` statement rather than a `ternary`

```
if (!barnAudio.playing()) barnAudio.play()
```

Reply:
okay, thanks for the feedback. i will be changing these.